extensions/lifecycle/util/rules.js

```diff
@@ -201,7 +201,7 @@ function rulesToParams(versioningStatus, currentDate, bucketLCRules, bucketData,
  * @returns {string[]} formatted supported lifecycle rules
  */
 function getFormattedSupportedLifecycleRules() {
-   return supportedLifecycleRules.map(rule => {
+   return config.extensions.lifecycle.supportedLifecycleRules.map(rule => {
         if (rule === 'noncurrentVersionTransition') {
```

Comment:
I think it's better if we pass the supported rules as an argument.
This is used in the bucket processor where lifecycle config is available [here](https://github.com/scality/backbeat/blob/0a32e66773027f99b5057bf9159bf1ec1f5fc3c0/extensions/lifecycle/bucketProcessor/LifecycleBucketProcessor.js#L84)

We then should also change the function name to `formatSupportedLifecycleRules`.

Reply:
done here : https://github.com/scality/backbeat/pull/2548/commits/08e817a420a339682e58d7d7776932e2d98e5f75#diff-c560baf5a4758706eaf47282c6fd3b381b38bc8b0eb355aa3983e7e675a8fc09R203-R204